src/Utils.ts

```diff
@@ -49,6 +49,14 @@ export default class Utils {
     });
   }
 
+  public static isValidHttpsUrl(url: string): boolean | string {
```

Comment:
Hi @VelinGeorgiev, kinda like the idea of having a common way to check these things, but as you wish in can be changed to inline :)

Reply:
My thinking is that the common way is already there in the Nodejs url libraries Yannick. I wouldn't create an extra method in the Utils class because the more stuff we add there the more difficult to maintain and understand this file will become. @waldekmastykarz, @garrytrinder  what do you think?